pkgdown/templates/header.html

```diff
@@ -1,45 +1,82 @@
-<head>
-<meta charset="UTF-8">
-<title>Untitled Document</title>
-<link href="extra.css" rel="stylesheet" type="text/css" media="screen" />
-<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
-	<!--[if lt IE 9]>
-    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-   <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-  <![endif]-->
- 
-
-  <!-- Google Tag Manager -->
-<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-})(window,document,'script','dataLayer','GTM-TKQR8KP');</script>
-<!-- End Google Tag Manager -->
```

Comment:
Does the new page still have google tags? Having trouble locating in the changed text

Reply:
I added that tag stuff to a new file which is injected into the header. Locally when I build the pkgdown site with the new code it's in there. Once we merge this PR, we can double-check that everything's working as expected on both GitHub and GitLab pages.